styles/header.css

```diff
@@ -66,15 +93,31 @@
  * other elements during opening of the menu makes them jump around, so we also
  * have to disable animation of opening/closing the nav. */
 .navbar-collapse hr {
-  opacity: 1;
+  opacity: 0.5;
+  border-color: var(--bs-navbar-color);
   margin-left: calc(var(--bs-gutter-x) * -0.5);
 
+  .bg-dark & {
+    opacity: 0.75;
+  }
+
   @media (width >= 576px) {
-    margin-left: calc((((100vw - 540px) / 2) * -1) - (var(--bs-gutter-x) * 0.5));
+    margin-left: calc(
+      (((100vw - 540px) / 2) * -1) - (var(--bs-gutter-x) * 0.5)
+    );
   }
 }
 
 .masthead {
+  --bs-link-color-rgb: var(--stanford-black-rgb);
+  --bs-link-hover-color-rgb: var(--stanford-black-rgb);
+
+  &.bg-dark,
+  .bg-dark & {
```

Comment:
Why do we need both of these variants?  Can you explain with a comment in the code?